website/docs/sdk-reference/dotnet.md

```diff
@@ -689,6 +689,11 @@ IConfigCatClient client = ConfigCatClient.Get("#YOUR-SDK-KEY#", options =>
 });
 ```
 
+:::info
+The .NET SDK supports [shared caching](/docs/advanced/caching/#shared-cache) since v8.1.0. As for other SDKs, please check their docs for the minumum versions
```

Comment:
```suggestion
The .NET SDK supports [shared caching](/docs/advanced/caching/#shared-cache) since v8.1.0. As for other SDKs, please check their docs for the minimum versions
```

Reply:
Nice catch! 👍 Thanks for the correction.